content/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -127,8 +135,8 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
     if (isActive && expandedImage !== undefined) {
       setImageIdInURL(expandedImage?.id || '');
     } else {
-      // clear the url of the fragments and also removes the # symbol
-      setImageIdInURL('');
+      // clear the url of the fragments and includeing the # symbol
```

Comment:
np typo `including`
But actually is this 
"clear the URL of the fragments, including the # symbol"?